deployments/helm-chart/templates/controller-daemonset.yaml

```diff
@@ -26,15 +26,15 @@ spec:
       terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
 {{- if .Values.controller.nodeSelector }}
       nodeSelector:
-{{ toYaml .Values.controller.nodeSelector | indent 8 }}
+{{ toYaml .Values.controller.nodeSelector | trimSuffix "\n" | indent 8 }}
```

Comment:
here and in the other templates, I don't see a need to use `trimSuffix`.  that unnecessary newline doesn't hurt.

Reply:
agreed. let's stay consistent with the rest of the template